search/src/main/kotlin/com/ritense/search/web/rest/SearchFieldV2Resource.kt

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2015-2022 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.search.web.rest
+
+import com.ritense.search.domain.SearchFieldV2
+import com.ritense.search.service.SearchFieldV2Service
+import javax.validation.Valid
+import org.springframework.http.MediaType
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+
+@RequestMapping("/api/v1/search/field", produces = [MediaType.APPLICATION_JSON_VALUE])
+class SearchFieldV2Resource(
+    private val searchFieldV2Service: SearchFieldV2Service
+) {
+
+    @PostMapping("/{ownerId}")
```

Comment:
When I call this endpoint:
```
POST http://localhost:8080/api/v1/search/field/leningen
{
    "title": "Start date",
    "key": "startDate",
    "path": "doc:startDate",
    "dataType": "date",
    "matchType": "exact"
}
```

I get this error:
```
com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `com.ritense.search.domain.DataType` from String "date": not one of the values accepted for Enum class: [BOOLEAN, NUMBER, TEXT, DATE, TIME, DATE_TIME]
```

Reply:
I think thats because date DataType needs to be filled with all caps like DATE